OpenRA.Game/Network/ReplayRecorderConnection.cs

```diff
@@ -50,8 +50,7 @@ void StartSavingReplay(byte[] initialContent)
 			var mod = Game.ModData.Manifest.Mod;
 			var dir = Platform.ResolvePath("^", "Replays", mod.Id, mod.Version);
 
-			if (!Directory.Exists(dir))
-				Directory.CreateDirectory(dir);
+			Directory.CreateDirectory(dir);
```

Comment:
I don't know about that... The docs for .Net Framework and Mono both say that it does nothing if directory already exists:
https://msdn.microsoft.com/en-us/library/54a0at6s%28v=vs.110%29.aspx
http://docs.go-mono.com/?link=M%3aSystem.IO.Directory.CreateDirectory%28System.String%29